.github/workflows/rust.yml

```diff
@@ -121,20 +75,17 @@ jobs:
       - name: Checkout source code
         uses: actions/checkout@v2
 
-      - name: Remove default members
-        run: sed '/default-members/d' -i ${{ matrix.directory }}/Cargo.toml
-
       - name: Install Rust
-        id: toolchain
         uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: ${{ matrix.toolchain }}
           override: true
-          components: clippy
 
-      - name: Install tools
-        uses: taiki-e/install-action@cargo-hack
+      - name: Install cargo-make
+        run: |
+          cargo install --debug cargo-quickinstall
```

Comment:
This seems... strange, is it that valuable? It adds another dependency and way for things to go wrong in my eyes. Is quickinstall at least re-used for the other install tasks in cargo-make?

Reply:
Using `cargo-quickinstall` is a very fast way for installing `cargo-make` which avoids recompiling `cargo-make`. `--debug` because it's even faster.

Reply:
I don't think it's re-used in `cargo-make`

Reply:
It feels a little strange for us to install this tool to use it once.. I can't imagine the speed-up is that consequential

Reply:
It is because it avoids compiling `cargo-make`. Compiling `cargo-quickinstall` is faster by a lot

Reply:
I understand that but if we're installing other cargo tools like clippy normally and compiling them ourselves, then my point is we're benefiting from this speed-up **only for** compiling `cargo-make`. And we're adding the risk of more external dependencies, relying on other releases, etc. This does not seem like a worthwhile trade-off to me.

Reply:
That's a fair point. I don't have a strong opinion on that so I changed this to compile it directly: 81e5addb5dbec81996445e35b7f360bce5df2e0b